documentation/modules/deploying/proc-using-access-operator.adoc

```diff
@@ -0,0 +1,130 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-access-operator.adoc
+
+[id='proc-using-access-operator-{context}']
+= Using the Access Operator (early access)
+
+[role="_abstract"]
+This procedure shows how to use the Access Operator to manage access to a Kafka cluster.
+The Access Operator simplifies access management by creating a binding `Secret` containing the connection details for a Kafka cluster and optional user credentials.
+
+Kafka instances and Kafka users are specified in a `KafkaAccess` resource.
+Optionally, a listener to connect to the Kafka cluster can also be specified. 
+The operator will look up the Kafka instance and users specified in the `KafkaAccess` resource to create a secret with the details required for connection.
+Any authentication credentials required by a user to connect to Kafka are also added to the secret. 
+If no listener is specified it will choose according to these rules: 
+
+* If only one listener is configured in the `Kafka` resource, that listener is selected.
+* If multiple listeners are configured, the operator selects one with the appropriate security based on tls and authentication properties in the `Kafka` and `KafkaUser` resources.
```

Comment:
should we have singol or double quotes around tls and authentication .... they are properties